apps/game_client/lib/game_client_web/live/pages/board/show.ex

```diff
@@ -14,10 +14,11 @@ defmodule GameClientWeb.BoardLive.Show do
     {:ok, game_socket_handler_pid} =
       GameClient.ClientSocketHandler.start_link(self(), gateway_jwt, player_id, game_id)
 
-    mocked_board_width = 2000
-    mocked_board_height = 2000
-    backend_board_size = 10_000
+    mocked_board_width = 3000
+    mocked_board_height = 3000
+    backend_board_size = 15_000
     back_size_to_front_ratio = backend_board_size / mocked_board_width
+    map_radius = round(5520 / back_size_to_front_ratio)
```

Comment:
Shouldn't this hardcoded 5520 be read from the JSON config instead?
What about the other values? Why those?

Reply:
the other 3 values besides `map_radius` are "configurations" for the game client size, this 5520 can be taken from the config file since  it's the radius from the backend

Reply:
done!